examples/powertools-examples-core/cdk/app/src/main/java/helloworld/App.java

```diff
@@ -43,6 +29,18 @@
 import software.amazon.lambda.powertools.tracing.Tracing;
 import software.amazon.lambda.powertools.tracing.TracingUtils;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static software.amazon.lambda.powertools.metrics.MetricsUtils.metricsLogger;
```

Comment:
I think it does not respect the checkstyle format. Make sure to update your branch with main and apply the checkstyle formatting in your IDE.

See https://github.com/aws-powertools/powertools-lambda-java/blob/docs/contributing/CONTRIBUTING.md#code-formatting

Reply:
Done.
We would need to do the same for other examples, though as well:
https://github.com/aws-powertools/powertools-lambda-java/blob/0e76f04f1946dc3cabb842fe4303ac122b67b563/examples/powertools-examples-idempotency/src/main/java/helloworld/App.java#L41-L40